### PR TITLE
Add Discord4J

### PR DESCRIPTION
**Pull request by Tais993**

### CODE REVIEW

**Comment by advaith1** [comment]
according to [releases](https://github.com/Discord4J/Discord4J/releases) d4j supports slash cmds, buttons, and selects, so this seems inaccurate?

**Comment by Tais993** [comment]
You're right, thought those were on the development branche

File: libs.ts [hunk 1]
```diff
@@ -69,6 +69,22 @@ export const libs: Lib[] = [
 			url: 'https://github.com/DV8FromTheWorld/JDA/pull/1769'
 		}
 	},
+	{
+		name: 'Discord4J',
+		url: 'https://github.com/Discord4J/Discord4J',
+		language: 'Java',
+		apiVer: '6 stable, 8 dev',
+		gwVer: '6 stable, 8 dev',
+		slashCommands: 'Dev Version',
```
Comment (Tais993) [inline libs.ts:78]:
```suggestion
		slashCommands: 'Yes',
```

File: libs.ts [hunk 2]
```diff
@@ -69,6 +69,22 @@ export const libs: Lib[] = [
...
+		buttons: 'Dev Version',
```
Comment (Tais993) [inline libs.ts:79]:
```suggestion
		buttons: 'Yes',
```

File: libs.ts [hunk 3]
```diff
@@ -69,6 +69,22 @@ export const libs: Lib[] = [
...
+		selectMenus: 'Dev Version',
```
Comment (Tais993) [inline libs.ts:80]:
```suggestion
		selectMenus: 'Yes',
```

**Comment by Tais993** [comment]
Just saw [this issue](https://github.com/Discord4J/Discord4J/issues/958) and [this pull](https://github.com/Discord4J/discord-json/pull/86)
Should I link the issue or the pull?

I'd assume issue since it gives more general information

**Comment by advaith1** [comment]
yeah put the issue

File: libs.ts [hunk 4]
```diff
@@ -69,6 +69,22 @@ export const libs: Lib[] = [
...
+		url: 'https://github.com/Discord4J/Discord4J',
+		language: 'Java',
+		apiVer: '6 stable, 8 dev',
+		gwVer: '6 stable, 8 dev',
+		slashCommands: 'Yes',
+		buttons: 'Yes',
+		selectMenus: 'Yes',
+		threads: {
+			text: 'Has a PR',
+			url: 'https://github.com/Discord4J/Discord4J/pull/923'
```
Comment (Tais993) [inline libs.ts:83]:
```suggestion
			url: 'https://github.com/Discord4J/Discord4J/issues/958'
```

**Pull request merged**